corehq/apps/app_manager/util.py

```diff
@@ -545,3 +545,18 @@ def change_unique_id(form):
                 jsonpath_update(reference, id_changes[reference.value])
 
     return app_source
+
+
+def append_update(dict_, new_dict):
```

Comment:
Yeah, it's here for the test. I couldn't run the test when it was nested, and it felt wrong in suite_xml.py when it wasn't nested. The test isn't amazingly useful, so I'm happy to put it back.